test/clj/rems/api/test_end_to_end.clj

```diff
@@ -367,7 +373,8 @@
                                                   :form/title "e2e"
                                                   :form/fields [{:field/type :text
                                                                  :field/title {:en "text field"
-                                                                               :fi "tekstikenttä"}
+                                                                               :fi "tekstikenttä"
+                                                                               :sv "textfält"}
```

Comment:
A bit inconsistent that above all the titles are "text field" for all languages and here translated. If there aren't two many in the file maybe change all to translated ones?

Reply:
fair enough :)